news/2023/2023-02-09-Skin-of-the-Year-2022.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Skin of the Year 2022: Voting Open"
+date: 2023-02-09 09:00:00 +0000
+---
+
+Do you think you got what it takes to select the BEST skins of 2022? Let us know by casting your vote!
+
+![](/wiki/shared/news/2023-02-01-Skin-of-the-Year-2022/SOTY-2022-banner.jpg)
+
+Four score and seven years ago (that being 2007), skinning was first introduced to the then newly released EBA simulator called osu!. Ever since, a sprawling and ever so creative community has been constantly expanding and evolving, many thousands of skins have been uploaded during the past 15 years, many styles and iterations of skins have come and gone, but the community never had a "Skin of the Year" competition. Instead of taking a moment to ponder at why that is, let's instead focus on the actual voting in this first-of-a-kind skin competition!
```

Comment:
What does "Four score and seven years ago" mean? I've read this about 10 times and cannot understand what you're trying to say.

Reply:
Also osu! was never really an EBA simulator. Might be better to say "Ouendan simulator".

Reply:
removed the four score part and changed EBA into Ouendan via https://github.com/ppy/osu-wiki/pull/8773/commits/2a566039435bc460fb51056c4f7d4e87dbaa43dc

Reply:
for the record, it's a reference to the gettysburg address and is a decently common literary inference to 'something that happened a while ago'